### PR TITLE
Add common styles 

### PR DESCRIPTION
**Pull request by StephenAraka**
### what does this PR do?
- add local font files
- add commonly used styles to index.css
- also remove styles from App.css (realized they were not being used)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  display: block;
+}
+
+input[type="text"],
+input[type="search"],
+input[type="email"],
+input[type="tel"],
+input[type="password"] {
+  box-sizing: border-box;
+}
```
Comment (Mubangizi) [inline src/index.css]: Just for clarity, Class names should be created with Pascal Case @StephenAraka

**Comment by Mubangizi** [comment]
> Good Stuff Just Casing Thanks

**Comment by Mubangizi** [comment]
Sorry for that

**Comment by StephenAraka** [comment]
> image
> @StephenAraka how many files are changed on this PR? Welcome to Software Development 101.

10/14 of the 

> image
> @StephenAraka how many files are changed on this PR? Welcome to Software Development 101.

12/15 of those files are local font files. not code. but thanks.

**Comment by StephenAraka** [comment]
> We had a consensus and resolved that `.eslintrc.json` should remain the way it is, we just need to follow best practices by changing our component files to `.jsx`.

I do not concur with this. Rules are meant to enforce best practices. Not everyone knows these best practices.

**Comment by StephenAraka** [comment]
> Good Stuff Just Casing Thanks

Hey Allan, the PascalCasing only applies to class names within a component. not the general class names.

File: src/index.css (hunk 1 above)
Comment (StephenAraka) [reply src/index.css]: > Just for clarity, Class names should be created with Pascal Case @StephenAraka

Not all class names @Mubangizi. Only component class names

**Pull request merged**